composer.json

```diff
@@ -7,8 +7,18 @@
         "php": ">=5.4.0",
         "react/promise": "~2.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.5"
```

Comment:
:+1: on discussion it as I agree with @assertchris that it would make it easier for anyone to jump in